backend/src/tasks/notifications.ts

```diff
@@ -0,0 +1,62 @@
+import { Handler } from 'aws-lambda';
+import { sendEmail } from '../api/helpers';
+import { connectToDatabase, Organization, Role } from '../models';
+import S3Client from './s3-client';
+
+const sendReportEmail = async (email: string, organization?: Organization) => {
+  await sendEmail(
+    email,
+    'New Report Available',
+    `Hi there,
+
+The latest report for ${
+      organization?.name ? `is now available on ` : ''
+    }Crossfeed. To view the report, sign on at ${process.env.FRONTEND_DOMAIN}
+
+Crossfeed access instructions:
+
+For more information on using Crossfeed, view the Crossfeed user guide at https://docs.crossfeed.cyber.dhs.gov/user-guide/quickstart/. 
+
+If you encounter any difficulties, please feel free to reply to this email (or send an email to ${
+      process.env.CROSSFEED_SUPPORT_EMAIL_REPLYTO
+    }).`
+  );
+};
+
+export const handler: Handler = async (event) => {
+  await connectToDatabase();
+  console.log('Running notifications check...');
+
+  // Get CISA organization[s] only
+  const organizations = await Organization.find({
+    where: {
+      name: 'CISA'
+    }
+  });
+
+  for (const org in organizations) {
+    const client = new S3Client();
+    // pull reports list
+    const reportsList = await client.listReports(organizations[org].id);
+    // find org admin roles for each organization
+    const roles = await Role.find({
+      relations: ['user', 'organization'],
+      where: {
+        role: 'admin',
+        organization: organizations[org]
+      }
```

Comment:
We should make sure we include `approved: true` in the query.

Reply:
added in [b1c2384](https://github.com/cisagov/crossfeed/pull/1709/commits/b1c238472c6c26f5f3a21a5bd9f5ab24730efffe)